### DISCUSSION TITLE
I ran the default script.But know the cast is not working

### DISCUSSION THREAD

**Discussion by Leopotter1218** [Raphire/Win11Debloat#145] [Q&A]
I used a amd device so tried reinstalling the drivers for igpu and wifi card. My wifi card supports cast but this script is the only script that I ran. 
Is there any possibility that the script is blocking the cast option?![8c3245c9-fdcc-444d-ac0e-30c4c4777744.jpg](https://github.com/user-attachments/assets/9eec43f9-bcdc-4ad0-a5aa-9c35be4eba5c)

**Comment by Raphire** [comment]
Heya,

The script does not interact with anything that could prevent you from casting your screen, so I'm unsure what could cause this issue. I do know that Screen casting seems extremely spotty, my work laptop refuses to cast while my personal laptop works fine.
